src/resources/views/crud/list.blade.php

```diff
@@ -155,8 +160,22 @@
 @endsection
 
 @section('after_scripts')
-	@include('crud::inc.datatables_logic')
-
+    @include('crud::inc.datatables_logic')
+
+  <script type="text/javascript">
```

Comment:
Ugh... now that I see the CSS&JS solution implemented @pxpm - I'm even more inclined to go with Solution 3 😆 I don't mind the CSS, but it looks a little dirty to have this JS to prevent the click on ```a.unstyled-anchor```. What can I say?... You were right, I was wrong.

**Let's go with Solution 3 please, where we allow the developer to choose the wrapping element, but default to ```a```**. This should provide maximum flexibility AND be a cleaner implementation.

Let me know if you'd rather I do it, it was never my plan to get help with this feature - specifically because I knew there would be a lot of rewrites.

Reply:
Did it. 

Indeed, it's just hacky, and probably would fail in some browsers etc. Solution 3 is more general and provides all the flexiblity we could ever want.